### PR TITLE
Monitoring container ansible playbook

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

File: playbooks/adhoc/deploy_monitoring_containers/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+- name: Setup hosts
+  hosts: localhost
+  gather_facts: no
+  user: root
+  tasks:
+  - name: build inven
```
Comment (twiest) [inline playbooks/adhoc/deploy_monitoring_containers/deploy.yml]: build inventory

**Comment by mwoodson** [comment]
:+1:

File: playbooks/adhoc/deploy_monitoring_containers/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        owner: root
+        group: root
+        mode: 0644
+    register: zagg_systemd
+
+  - name: reload systemd
+    command: /usr/bin/systemctl --system daemon-reload
+    when: pcp_systemd.changed or zagg_systemd.changed
+
+  - name: pasue for a few seconds
```
Comment (twiest) [inline playbooks/adhoc/deploy_monitoring_containers/deploy.yml]: pause

Comment (twiest) [inline playbooks/adhoc/deploy_monitoring_containers/deploy.yml]: In the future, we'll want to make this conditional on if things changed. We can do that as a separate PR.

**Pull request merged**